apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -236,6 +238,34 @@ const Ticket = new GQLListSchema('Ticket', {
             knexOptions: { isNotNullable: true }, // Relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
         },
+        propertyAddress: {
+            schemaDoc: 'Address of property, which synced with property and displayed, if property is deleted',
+            type: Text,
+            isRequired: true,
```

Comment:
It's should be a read-only field. You should not have access to change this by API.

Reply:
see comments above